### PR TITLE
Adding newly generated Vnet files

### PR DESCRIPTION
**Pull request by dnayantara**
Adding the new regenerated Vnet files. Swagger changes in the pull request - https://github.com/Azure/azure-rest-api-specs/pull/1492 

This adds optional parameters so the current tests are successful as they are.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dnayantara** [comment]
Apologies for the multiple commits for numerous merges. I will make sure it does not happen going forward.

**Review by jaredmoo — APPROVED** [review]

**Comment by dnayantara** [comment]
I dont understand. Yesterday it said all checks had passed. Why is build failing now? :(

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

File: src/SDKs/SqlManagement/Management.Sql/Generated/Models/VirtualNetworkRule.cs [hunk 1]
```diff
@@ -33,15 +33,22 @@ public VirtualNetworkRule()
...
         /// <param name="type">Resource type.</param>
-        /// <param name="virtualNetworkSubnetId">The resource ID of the virtual
-        /// network subnet</param>
-        public VirtualNetworkRule(string id = default(string), string name = default(string), string type = default(string), string virtualNetworkSubnetId = default(string))
+        /// <param name="ignoreVnetPrivateAccessConfiguration">Create firewall
+        /// rule before the virtual network has private access enabled.</param>
+        /// <param name="state">Virtual network rule state. Possible values
+        /// include: 'Initializing', 'InProgress', 'Ready', 'Deleting',
+        /// 'Unknown'</param>
+        public VirtualNetworkRule(string virtualNetworkSubnetId, string id = default(string), string name = default(string), string type = default(string), bool? ignoreVnetPrivateAccessConfiguration = default(bool?), string state = default(string))
```
Comment (shahabhijeet) [inline src/SDKs/SqlManagement/Management.Sql/Generated/Models/VirtualNetworkRule.cs]: @dnayantara you already have tests for the newly added parameter?
The description of the PR was confusing?
Yes the existing tests will work, but what about testing the new parameter?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/SqlManagement/Management.Sql/Generated/Models/VirtualNetworkRule.cs (hunk 1 above)
Comment (dnayantara) [inline src/SDKs/SqlManagement/Management.Sql/Generated/Models/VirtualNetworkRule.cs]: The code is not deployed in production yet. I will update the tests once that happens. The client code has been internally tested successfully.

**Comment by dnayantara** [comment]
The backend code is not yet in production. We cannot publish the nuget package with these changes unless everything is in production.

**Pull request closed without merging**